lib/utils/report.mjs

```diff
@@ -115,6 +125,45 @@ export default function report(problem) {
 	result.warn(warningMessage, warningProperties);
 }
 
+/**
+ * @param {Problem} problem
+ */
+function problemRangeDeprecations(problem) {
+	// TODO: `config.quietDeprecationWarnings` is not available in `problem`
```

Comment:
[question] Is it possible to use `config.quietDeprecationWarnings` from `result.stylelint`?

Reply:
Unfortunately not.
`quietDeprecationWarnings` is not included there: https://github.com/stylelint/stylelint/blob/3a903800248fcccd4968e8e0dc4a76a4d8b88ff4/types/stylelint/index.d.ts#L89-L116

As I understand it, the value does exist higher up the stack, but isn't passed to the `config` object present on `result.stylelint`

Reply:
Ah, from `config.quietDeprecationWarnings`, I misunderstood `quietDeprecationWarnings` would be in `Config`, sorry.

Instead, is it possible to add `quietDeprecationWarnings` to `StylelintPostcssResult` like `quiet`?

Reply:
I've added `quietDeprecationWarnings` like `quiet` and added test coverage.